.github/workflows/publish-results.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish results
+
+on:
+  workflow_run:
+    workflows: ["Benchmark"]
+    types:
+      - completed
+
+jobs:
+  publish_results:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install python
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.10'
+      - name: Install Requirements
+        run: pip install -r requirements.txt
+      - name: Git config
+        run: |
+          git config --global user.name "gh-pages bot"
+          git config --global user.email "gh-pages-bot@benchmarks.com"
+      - name: Publish results
+        run: |
+          asv gh-pages --rewrite
+      - name: Commit results
```

Comment:
trying to run `asv gh-pages` directly gave me an error saying that gh-pages is behind origin and I need to pull first, so I added the `--rewrite` option which makes sure that only one commit exists in the branch, instead of adding a new commit. This could be a problem, so I have changed it and added a `git fetch` before pushing